ra_aid/agent_utils.py

```
@@ -10,9 +10,11 @@
 import time
 from typing import Optional
 
+from ra_aid.project_info import get_project_info, format_project_info, display_project_status
+
 from langgraph.prebuilt import create_react_agent
 from ra_aid.agents.ciayn_agent import CiaynAgent
-from ra_aid.agents.ciayn_agent import CiaynAgent
+from ra_aid.project_info import get_project_info, format_project_info, display_project_status
```

Ruff (0.8.2)

Redefinition of unused `get_project_info` from line 13

Remove definition: `get_project_info`

(F811)